.travis.yml

```diff
@@ -4,9 +4,9 @@ jdk:
 env:
   global:
     # Remember to keep this in sync with config.properties
-    - compileSdkVersion=23
-    - minSdkVersion=17
-    - targetSdkVersion=19
+    - compileSdkVersion=25
```

Comment:
`toolsVersion` and `compileSdkVersion` do not currently match `config.properties`

Which is probably why the Travis build is failing...